### ISSUE TITLE
Gerar certificados para todas as participantes, participantes de tutorial, ministranes,voluntárias e organização

### ISSUE THREAD

**Issue by cecivieira** [pythonbrasil/pybr2021-org#216]
### Descrição
Gerar certificados para todas as pessoas participantes do evento e voluntárias, além de enviá-los por e-mail.

### Requisitos para a atividade ser considerada como concluída

- [x] #332
- [X] #333
- [x] #334
- [ ] #335
- [ ] #336

**Comment by cecivieira** [comment]
Todas as pessoas inscritas no EventBrite receberão certificado de participante. #266 

Voluntárias e palestrantes também receberão

**Comment by gabubellon** [comment]
> Dá pra criar role para as pessoas do sábado de manhã, nem que seja pra todos os inscritos, mas teve gente que se inscreveu depois do tutorial acabar

Então @alynnefs  para os inscritos não precisa, e só para quem não tem a role
Vamos ter que decidir se damos certificados para inscritos ou pra quem tem role

TODO: Adicionar carimbo de data-hora na inscrição

**Comment by gabubellon** [comment]
Crieie issos em paraleo para isso

**Comment by alynnefs** [comment]
Posso marcar o de participantes como enviado? Se não me engano, só faltam os certificados que pedem a duração do evento

**Comment by gabubellon** [comment]
@alynnefs Pode, mas os que pedem hora fica beeem por ultimo

**Comment by cecivieira** [comment]
@gabubellon foram enviados os certificados para as ministrantes de tutoriais e palestras?

**Comment by gabubellon** [comment]
@cecivieira Não, nem gerados foram precisamos ter a lista de pessoas e gerar

**Comment by cecivieira** [comment]
@gabubellon Quais os dados necessários? Nome, título e data da palestra?

**Comment by gabubellon** [comment]
Se tiver mais de uma palestra/tutorial, tem que duplicar a info

**Comment by rougeth** [comment]
@cecivieira Tem demanda para esses certificados?

**Comment by cecivieira** [comment]
@rougeth não sei, pois não tô acessando o freshdesk.
Tô considerando essa atividade para fechar o ciclo desse evento.
